addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -241,12 +241,13 @@ $input-border-radius: o-website-value('input-border-radius') !default;
 $input-border-radius-lg: o-website-value('input-border-radius-lg') !default;
 $input-border-radius-sm: o-website-value('input-border-radius-sm') !default;
 
-$input-bg: var(--o-input-bg, #{$light}) !default;
-$input-border-color: var(--o-input-border-color, #{$light}) !default;
-$input-color: color-contrast($light) !default;
-$input-focus-bg: $body-bg !default;
-$input-focus-color: $body-color !default;
-$input-placeholder-color: var(--o-input-placeholder-color, #{rgba($input-color, .5)}) !default;
+$input-bg: o-website-value('input-background-color') or o-color('o-cc1-bg') !default;
+$input-disabled-bg: rgba($input-bg, .4) !default;
```

Comment:
Hey @stefanorigano 👋 

There's no difference between regular inputs and disabled inputs when the background color is not set.
<img width="1201" alt="Capture d’écran 2023-10-25 à 10 30 20" src="https://github.com/odoo/odoo/assets/80679690/b67ce6f5-a391-465f-b37e-07968fd8aa87">

Maybe use opacity on the element instead of only on the background? What do you think ?

Reply:
We have to block the user from selecting transparent and semi-transparent bg.
It conflicts with the disabled state, but also with the text color computation (in you example the text is readable just because the bg is white)